src/content/concepts/under-the-hood.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Under The Hood
+sort: 1
+contributors:
+  - smelukov
+---
+
+> This section describes webpack internals and useful for plugin developers
+
+The bundling is a function that takes some files and emits others.
```

Comment:
What do you mean?

Reply:
It's a very basic definition of bundling

Reply:
```suggestion
Bundling is a function that takes some files and emits others.
```